src/debugger/cli-debugger.c

```diff
@@ -3,6 +3,7 @@
  * This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+#include "mgba-util/common.h"
```

Comment:
This doesn't belong here. If <mgba-util/common.h> isn't in cli-debugger.h then it belongs there instead.

Reply:
reverted